css-basics/css-basics_pokemon-page/css/styles.css

```diff
@@ -57,24 +76,31 @@ section {
 - The right border should end with the last character of the title. Hint: If you don't know how to do this, think about the display property.
 - Bonus: The color of the border of "Favourite Pokémon" should be "orange". Hint: You may need another CSS rule.
 */
-
-.subtitle {
-  /* your solution */
+/* your solution */
+.subtitle { 
+  
+  padding: 10px 20px; 
+  border-left: 3px solid gold;
+  border-right: 3px solid gold;
+  display: inline-block;
 }
-
-/* ------- List ------- */
+ 
 
 /* Task 7: Last but not least! We want to style our list with our favorite Pokémon. Which selector do we need? And also add values for the padding. */
 
-/* add selector */
- {
+ ul{
   list-style: "🧡";
-  padding-left: /* add value */ ;
-}
+  color: rgb(33, 25, 143);
+  padding-left: 30px;
+  }
```

Comment:
There seems to be a bit of indentation issues here. Not a big problem but if you have "prettier" set as your default formatter for VSCode, the indentation should fix itself automatically upon saving a file :)